release-notes/3.0/preview/api-diff/preview8/.Net/standalone-packages/3.0-preview8-standalone-packages_System.ComponentModel.Composition.Primitives.md

```diff
@@ -0,0 +1,97 @@
+# System.ComponentModel.Composition.Primitives
```

Comment:
AFAIK we didn't do anything with MEF v1 in P8. Why does this show up now?

Reply:
Agreed. These shouldn't be in the diff (especially between p7/p8). The previously shipped NuGet package (during the 2.1 time frame) already has most (if not all) of these APIs.
https://www.nuget.org/packages/System.ComponentModel.Composition/4.5.0